### PR TITLE
Feat/migrate off new releases

### PR DESCRIPTION
**Pull request by venkatamutyala**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -0,0 +1,393 @@
+[
+  {
+    "repo": "https://github.com/actions/setup-go",
+    "dependencies": [
+      "https://github.com/GlueOps/alertmanager",
+      "https://github.com/GlueOps/github-workflows"
+    ]
```
Comment (fernandoataoldotcom) [inline packages.json:7]: to make sure I understand the data structure, the dependencies are the things that depend upon the repo, not the dependencies of the repo?

**Review by fernandoataoldotcom — APPROVED** [review]
a few questions, nothing major

File: packages.json (hunk 1 above)
Comment (venkatamutyala) [reply packages.json:7]: yes. there is probably a better name for this. any thoughts?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+TOKEN = os.environ['GITHUB_PAT']
+with open('packages.json', 'r') as f:
+    # Load the JSON data from the file
+    data = json.load(f)
+
+# Extract unique repos
+repos = set()
+for item in data:
+    repos.add(item['repo'])
+    for dep in item['dependencies']:
```
[the remark this answers is not in the document]
Comment (venkatamutyala) [reply main.py:15]: All the github URLs are combined into one big set and then that set get's starred/subscribed too later in the code

**Pull request merged**